delocate/libsana.py

```diff
@@ -625,7 +625,7 @@ def search_environment_for_lib(lib_path):
 
     for location in potential_library_locations:
         if os.path.exists(location):
-            return location
+            return realpath(location)
```

Comment:
Would you mind changing the docstring above to say that we return the realpath of the first found location, or the realpath if not found.

Reply:
Fixed.